lib/CMakeLists.txt

```diff
@@ -134,17 +134,51 @@ target_include_directories(chiaki-lib PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/includ
 find_package(Threads REQUIRED)
 target_link_libraries(chiaki-lib Threads::Threads)
 
-find_package(PkgConfig REQUIRED)
-pkg_search_module(json-c REQUIRED json-c IMPORTED_TARGET)
-target_include_directories(chiaki-lib PRIVATE ${json-c_INCLUDE_DIRS})
-target_link_libraries(chiaki-lib PkgConfig::json-c)
+if (CHIAKI_IS_SWITCH)
+	find_library(JSONC_LIB json-c /opt/devkitpro/portlibs/switch/lib)
+	target_link_libraries(chiaki-lib ${JSONC_LIB})
+	target_include_directories(chiaki-lib PUBLIC "/opt/devkitpro/portlibs/switch/include/json-c")
```

Comment:
This can probably be improved, but it refuses to find this library unless I explicitly link to it here.

Reply:
does settings CPATH help?

Reply:
you can also just add to the cmake command cmake -DCMAKE_LIBRARY_PATH=/opt/devkitpro/portlibs/switch/lib -DCMAKE_INCLUDE_PATH=/opt/devkitpro/portlibs/switch/include/ ..

Reply:
you can also use cmake -DCMAKE_PREFIX_PATH=/opt/devkitpro/portlibs/switch/ ..